test/navnode.jl

```diff
@@ -29,7 +29,7 @@ pages = [
     "page6.md",
 ]
 doc = FakeDocument()
-doc.internal.pages = Dict(["page$i.md" => nothing for i=1:8])
+doc.internal.pages = Dict(["page$i.md" => nothing for i=1:8]) # NOTE (Compat): deprecated Dict syntax for 0.4 compatibility
```

Comment:
I think `Dict(map(i -> "page$i.md" => nothing, 1:8))` should work fine on all versions and it's hardly more verbose than the generator syntax anyway.